.eslintrc

```diff
@@ -1,34 +1,24 @@
-module.exports = {
+{
+    "extends": "airbnb-base",
     "env": {
-        "node": true,
         "browser": true,
-        "es6": true,
-        "jquery": true,
-        "jasmine": true,
-        "mocha": true
-    },
-    "extends": "ovh",
-    "rules": {
+        "jquery": true
     },
     "globals": {
         "_": true,
-        "$": true,
-        "JSURL": true,
         "angular": true,
+        "Chart": true,
+        "EprotectIframeClient": true,
+        "ipaddr": true,
+        "JSURL": true,
         "moment": true,
         "punycode": true,
+        "RandExp": true,
         "URI": true,
-        "Util": true,
-        "Chart": true,
-        "ipaddr": true,
-        "JustGage": true,
-        "filesize": true,
         "validator": true,
-        "Raven": true,
-        "RFB": true,
-        "WebUtil": true,
-        "RandExp": true,
-        "EprotectIframeClient": true,
         "zE": true
+    },
+    "rules": {
+        "no-bitwise": ["error", { "allow": ["~"] }]
```

Comment:
You can change all `~xxx.indexOf` to `-1 !== xxx.indexOf`, or else we never will get rid of `~`

Reply:
~ FTW!

Reply:
You're part of the problem @blaryjp